open-api/swagger.yaml

```diff
@@ -0,0 +1,1145 @@
+
+definitions:
+  v1observation:
+    type: object
+    properties:
+      subject:
+        $ref: '#/definitions/patient'
+      label:
+        type: string
+      value:
+        type: string
+
+  ontologyTerm:
+    type: object
+    properties:
+      name:
+        type: string
+      key:
+        type: string
+      fullName:
+        type: string
+      type:
+        type: string
+
+  jsonStudy:
+    type: object
+    properties:
+      id:
+        type: string
+      ontologyTerm:
+        type: object
+        properties:
+          name:
+            type: string
+          key:
+            type: string
+          fullName:
+            type: string
+          type:
+            type: string
+
+  hal+jsonStudy:
+    type: object
+    properties:
+      id:
+        type: string
+      _links:
+        type: object
+        properties:
+          self:
+            type: object
+            properties:
+              href:
+                type: string
+      _embedded:
+        type: object
+        properties:
+          ontologyTerm:
+            type: object
+            properties:
+              name:
+                type: string
+              key:
+                type: string
+              fullName:
+                type: string
+              type:
+                type: string
+                default: 'STUDY'
+              _links:
+                type: object
+                properties:
+                  self:
+                    type: object
+                    properties:
+                      href:
+                        type: string
+                  observations:
+                    type: object
+                    properties:
+                      href:
+                        type: string
+                  children:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        href:
+                          type: string
+                        title:
+                          type: string
+
+  hal+jsonStudys:
+    type: object
+    properties:
+      _links:
+        type: object
+        properties:
+          self:
+            type: object
+            properties:
+              href:
+                type: string
+      _embedded:
+        type: object
+        properties:
+          studies:
+            type: array
+            items:
+              $ref: '#/definitions/hal+jsonStudy'
+              
+  storageSystem:
+    type: object
+    properties:
+      id:
+        type: integer
+      name:
+        type: string
+      systemType:
+        type: string
+      url:
+        type: string
+      systemVersion:
+        type: string
+      singleFileCollections:
+        type: boolean
+  
+  fileLink:
+    type: object
+    properties:
+      id:
+        type: integer
+      name:
+        type: string
+      sourceSystem:
+        type: integer
+      study:
```

Comment:
Short case insensitive String identifying tranSMART study,usually study name, given during ETL, can be retrieved by /studies (I think both v1 and v2)